.github/workflows/ci.yml

```diff
@@ -56,10 +56,10 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ${{ env.CLCACHE_DIR }}
-          key: windows-2019-clcache-${{github.ref}}-${{github.sha}}
+          key: windows-vs2019-clcache-${{github.ref}}-${{github.sha}}
           restore-keys: |
-            windows-2019-clcache-${{github.ref}}-
-            windows-2019-clcache-refs/heads/main-
+            windows-vs2019-clcache-${{github.ref}}-
+            windows-vs2019-clcache-refs/heads/main-
```

Comment:
I see that #1325 was merged on 5 May, so based on this limited sample size of N=1, we have two months of uncorrupted runs